content/docs/deployment/ghpages/contents.lr

```diff
@@ -34,6 +34,13 @@ rules apply as for the [rsync deployment method :ref](../rsync/).  The HTTPS
 transport on the other hand accepts `--username` and `--password` which
 override the values in the URL.
 
+Example:
+
+```ini
+[servers.ghpages]
+target = ghpages:https://username:pasword@your-user/your-user.github.io
```

Comment:
I don't agree with this approach as we are basically advising users how to expose their credentials.

Reply:
Can you explain how to set up enviroment variables? Cause then that is not clear for me.

Reply:
https://www.getlektor.com/docs/deployment/travisci/